awx/ui_next/src/components/TemplateList/TemplateList.jsx

```diff
@@ -168,6 +169,13 @@ function TemplateList({ defaultParams, i18n }) {
     <AddDropDownButton key="add" dropdownItems={addDropDownButton} />
   );
 
+  const deleteDetailsRequests = {
+    [i18n._(t`Workflow Job Template Node`)]: async () =>
```

Comment:
it feels a little weird using translated strings as object keys here. Maybe consider restructuring it a bit to something like:

```
{
  template_node: {
    fetch: async () => {...},
    label: i18n._(t`Workflow Job Template Node`),
  }
}
```

(and maybe even change it into an array of those objects rather than an object of nested objects